### PR TITLE
added homepage redirect, intermediate cookie-clear start page, change …

### PR DESCRIPTION
**Pull request by raffcecco**
…home links/redirects, selenium fixes

### CODE REVIEW

File: webapp/src/main/resources/template/home.hbs [hunk 1]
```diff
@@ -1,62 +0,0 @@
-{{#partial "content" }}
```
Comment (aaronlong) [inline webapp/src/main/resources/template/home.hbs]: As the home page handlebars template has been removed the pre-compiling of the template in LambdaWarmUpBinder should also be removed.  Not sure how this deals with templates which are not there.  Probably better to remove it to be on the safe side :)

Comment (aaronlong) [reply webapp/src/main/resources/template/home.hbs]: On a further look warm up is enabled on NFT, PP and PRD.  So this should be changed to stop errors in those environments.

Comment (raffcecco) [reply webapp/src/main/resources/template/home.hbs]: Thanks for the heads-up. I'll look at that tomorrow.

Comment (raffcecco) [reply webapp/src/main/resources/template/home.hbs]: Done!

**Pull request merged**